src/components/views/elements/AppTile.js

```diff
@@ -269,29 +312,32 @@ export default React.createClass({
         }
 
         if (this.props.show) {
-            if (this.state.loading) {
+            if (this.state.initialising) {
                 appTileBody = (
                     <div className='mx_AppTileBody mx_AppLoading'>
                         <MessageSpinner msg='Loading...' />
                     </div>
                 );
             } else if (this.state.hasPermissionToLoad == true) {
+                const loadingElement = (
+                    <div className="mx_AppTileBody">
+                        <AppWarning
+                            errorMsg="Error - Mixed content"
+                        />
+                    </div>
+                );
                 if (this.isMixedContent()) {
-                    appTileBody = (
-                        <div className="mx_AppTileBody">
-                            <AppWarning
-                                errorMsg="Error - Mixed content"
-                            />
-                        </div>
-                    );
+                    appTileBody = loadingElement;
                 } else {
                     appTileBody = (
-                        <div className="mx_AppTileBody">
+                        <div className={this.loading ? 'mx_AppTileBody mx_AppLoading' : 'mx_AppTileBody'}>
+                            { this.loading && Element }
```

Comment:
Not sure what `Element` is

Reply:
Yeah, that's just weird ... not sure how that got committed. Looks like autocomplete cruft? Embarrassing. Thanks for catching that.